atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -137,6 +152,8 @@ def parse_image_build_config(self, config_file_name):
         section = 'image-build'
         for option in ('name', 'version', 'arches', 'target', 'install_tree'):
             value = config.get(section, option)
```

Comment:
It still would, but we're building a default config as an input to config parser which is dynamically generated. This check is to ensure that neither the config that was dynamically generated nor the config provided by user has somehow provided an empty value, such as:

```
install_tree =
```